docs/pages/includes/database-access/discovery-guide-brief.mdx

```diff
@@ -0,0 +1,20 @@
+Database discovery involves two components.
+
+The first, the Discovery Service, is responsible for watching your cloud
+provider and checking if there are any new databases or if there have been any
+modifications to previously discovered databases.
+
+The second, the Database Service, monitors the databases created by the
+Discovery Service and proxies communications between users and the database
+servers hosted by the cloud provider.
+
+In this guide, you will configure a Teleport Discovery Service to discover
+{{ cloud }}-hosted databases automatically.
+
+<Admonition type="note">
```

Comment:
I think we can remove this admonition, since it's not relevant for a user who is on v14 of the docs and following the guides that include this partial.

For a user on a previous version of the docs, they'll see the legacy instructions anyway.